ui/app/AppLayouts/Communities/CommunitiesPortalLayout.qml

```diff
@@ -137,14 +137,18 @@ StatusSectionLayout {
                     onClicked: Global.importCommunityPopupRequested()
                 }
 
-                StatusButton {
-                    id: createBtn
-                    objectName: "createCommunityButton"
-                    Layout.preferredHeight: 38
-                    verticalPadding: 0
-                    text: qsTr("Import existing community from Discord")
```

Comment:
Should we maybe the `import from discord` button commented here? so that it's easier to bring it back later.

Reply:
That's actually from the revert. This button doesn't exist on master.

Reply:
oh ok. I'd rather leave it in master ,but it's too late 😄